OWML.Launcher/SocketListener.cs

```diff
@@ -11,7 +11,8 @@ namespace OWML.Launcher
 {
     public class SocketListener
     {
-        private const int BufferSize = 1024;
+        private const string Separator = "\n--------------------------------";
+        private const int BufferSize = 262144;
```

Comment:
Sure this won't clog up *something*?

Reply:
worked fine with stack overflow trace (~90-100k symbols)